a = tuple(1, 2)[0], ValueError()
